gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/lru/LRUClockNode.java

```diff
@@ -79,4 +79,10 @@
    *
   public void resetRefCount(NewLRUClockHand lruList);
   */
+
+  /**
+   * Get in memory value for this node.
+   * @return value of this entry
+   */
+  public Object getValue();
```

Comment:
Better to change this to "isValueNull" like in RegionEntry. The reason why this was added in Gem layer instead of using "_getValue() == null" is for Gem off-heap which can cause trouble due to: a) for off-heap a _getValue() call will lead to deserialization of the entire value from off-heap to heap if it is not null, b) its an unretained call so they put an underscore to underline that fact but still there might be a race condition if caller is not careful

The isValueNull() and getValueAsToken() in general avoid this issue when the need is only to check against null or Token (removed/destroyed etc). For Gem off-heap they will just read the flags and return appropriately instead of deserializing the value. While we currently don't plan to use Gem off-heap but still its better to follow their conventions when touching Gem portions, so this can be a "boolean isValueNull()"